### PR TITLE
[NCP-3635] added a validation to reject use of an origin shield

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

**Review by burtonquantil — APPROVED** [review]
The English text sounds OK.

File: zh-Hans.json [hunk 1]
```diff
@@ -1607,6 +1607,7 @@
 	"property.shield.origin.details.lastUpdatedDate": "最后更新日期",
 	"property.shield.origin.details.name": "名称",
 	"property.shield.origin.details.region": "区域",
+	"property.shield.origin.errorModal.direction.connection": "仅当“直接连接”设置为“无直接”时才可使用屏蔽",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:1610]: Please change the text to:
只有当“直接连接”设置为“不直连，使用父缓存”时，才可以开启Shield

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]
Requested a change.

File: zh-Hans.json (hunk 1 above)
Comment (jungho73) [reply zh-Hans.json]: changed the text as recommended.
Thanks.

**Review by Jianghai-Quantil — APPROVED** [review]
Good to merge

**Pull request merged**